vaadin-rich-text-editor-flow-parent/vaadin-rich-text-editor-flow/src/main/java/com/vaadin/flow/component/richtexteditor/RichTextEditor.java

```diff
@@ -269,6 +272,32 @@ private String getDeltaValue() {
         return getElement().getProperty("value");
     }
 
+    /**
+     * Gets the list of colors used by the text color picker and background
+     * color picker controls of the text editor.
+     *
+     * @since 24.5
+     * @return the list of colors options
+     */
+    public List<String> getColorOptions() {
+        return JsonSerializer.toObjects(String.class,
+                (JsonArray) getElement().getPropertyRaw("colorOptions"));
+    }
+
+    /**
+     * Sets the list of colors used by the text color picker and background
```

Comment:
What format is required here to be considered a color? hex, hsl, name.. something else? 😀

Reply:
Added a mention that colors should be in HEX format (the web component popup uses that).
Note: the actual HTML value produced by Quill uses `rgb()` so the output looks like this:

```html
<p><span style="color: rgb(230, 0, 0);">Foo</span></p>
```